workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -32,17 +32,46 @@ public static void Build()
             try
             {
                 var args = CommandLineArgs.FromCommandLine();
-                var buildTargetArg = args.GetCommandLineValue("buildTarget", "local");
+                var buildEnvironmentArg = args.GetCommandLineValue("buildEnvironment", "local");
+                var buildTargetArg = args.GetCommandLineValue("buildTarget", "All");
```

Comment:
This is a breaking change as this is a public API - make sure to note in the changelog + upgrade guide. 

Also need to update https://docs.improbable.io/unity/alpha/modules/build-system/cli

Reply:
will update the PR description to do these^